doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -642,6 +642,10 @@ Multiprotocol Service Layer libraries
 Libraries for networking
 ------------------------
 
+* Added missing brackets that caused C++ compilation to fail in the following libraries:
+  * :ref:`lib_nrf_cloud_pgps`
+  * :ref:`lib_nrf_cloud_fota`
```

Comment:
```suggestion
* Added missing brackets that caused C++ compilation to fail in the following libraries:

  * :ref:`lib_nrf_cloud_pgps`
  * :ref:`lib_nrf_cloud_fota`
```
To avoid a doc build failure.